includes/class-tyche-plugin-deactivation.php

```diff
@@ -117,7 +117,7 @@ public function init_vars( $options ) {
 		public function plugin_action_links( $links ) {
```

Comment:
@nidhitatosaniya This file should be in the `includes/component/plugin-deacivation` folder.

Reply:
> @nidhitatosaniya This file should be in the `includes/component/plugin-deacivation` folder.

@nidhitatosaniya There's a mistake in the file name. It should rather be `includes/component/plugin-deactivation` folder.